### ISSUE TITLE
Weird antlers behaviour with simple output

### ISSUE THREAD

**Issue by danijelk** [statamic/cms#2675]
When putting our title in a simple logic expression the output gets escaped. Is this normal? Any way to stripslashes?

```
    {{ seo_title ? seo_title : title }}
    {{ title }}

    {{ title | dump }}
    {{ seo_title | dump }}
```

```
Let\'s work together 
Let's work together

"Let's work together"
null
```

**Comment by damcclean** [comment]
I've had a look into this one, looks like the issue is within [this regex condition on line 1003](https://github.com/statamic/cms/blob/master/src/View/Antlers/Parser.php#L1003) of the Parser.

The `$value` going into that method is correct but after the `while(` condition, `$value` is escaped as shown below.

**Comment by danijelk** [comment]
Thanks @damcclean seems like it's related to line 1054
`return var_export($value, true);`

But I just went back to good old {{ if else }} and that works ok as long as you don't do any logic within the {{ }} I iguess :)

**Comment by duncanmcclean** [comment]
Looks like the bot has already closed this issue but just FYI, #3123 has been merged, so it'll be fixed in the next 3.0 release.

**Comment by danijelk** [comment]
Saw it, nice work Duncan and thanks
